tests/tests.js

```diff
@@ -821,16 +969,15 @@ QUnit.test("onAuthStateChanged test", function(assert) {
 // And pretty much all of the same functions as localStorage (createCalendar, etc)
 
 QUnit.module( "CheckIt tests for clickHandlers", {
-  before: function() {
-    // prepare something once for all tests
-  },
   beforeEach: function() {
     // prepare something before each test
   },
   afterEach: function() {
     // clean up after each test
-  },
-  after: function() {
-    // clean up once after all tests are done
   }
 });
+
+
+// Have to have the same html in test html as my app html so that I can
```

Comment:
If you want to test that your buttons and stuff work, then yes probably. I'd caution against making sure that the correct functions are called though. Just make sure that the correct thing has happened. i.e. check that a checkmark has appeared, not that "create_checkmark" was called or whatever it is.